vic/vic_run/include/vic_def.h

```diff
@@ -60,7 +60,7 @@
 #define MIN_VEGCOVER    0.0001 /**< Minimum allowable vegcover fraction */
 
 /***** Define minimum and maximum values for model timesteps *****/
-#define MIN_SUBDAILY_STEPS_PER_DAY  4
+#define MIN_SUBDAILY_STEPS_PER_DAY  1
```

Comment:
I don't think I understand what you are trying to do.  The VIC image driver will not support a daily timestep. You will need to supply forcings at a maximum timestep of 6 hours. Is there a particular reason you want to run VIC at a daily timestep?

Previous versions of VIC disaggregated daily met forcings to subdaily on the fly using MTCLIM, but this cannot be done in any reasonable way in the image driver. We have also removed MTCLIM from the classic driver in VIC 5.  Relevant closed issues are #7 and #240.